contracts/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+# AVOID CHANGING REQUIRES: IT WILL BE UPDATED BY PYSCAFFOLD!
+requires = ["setuptools>=46.1.0", "setuptools_scm[toml]>=5", "wheel", "cairo-lang==0.9.1"]
```

Comment:
Why would we need `"cairo-lang==0.9.1"` for the PY publishing project?

Reply:
I initially wanted to compile `starknet-artifacts` when package is being built (on `setup.py` execution) and distribute this way. That would require cairo-lang + nile because `python build` creates separate venv. But unfortunately, there're such [bugs](https://github.com/OpenZeppelin/nile/issues/124) so artifacts with same names get overwritten by each other. So if we want to distribute artifacts to pypi we would need to first build `starknet-artifacts` with `hardhat-plugin` and the copy it into cairo package and after that start building python package

Reply:
Yeah, this was the original idea.

We don't need another env to compile the contracts, we use Hardhat for that which is already setup.
Ideally, we'd just need to package (c/p from source) those contracts and artifacts and publish them to PyPI via this simple Py env.

Reply:
Well ideally I would want Python package build be independent from node environment. But looks like that's hard to achieve. So will use that approach